src/components/screens/MainScreen.tsx

```diff
@@ -1,49 +1,41 @@
-import { BasicText } from 'components/Text'
-import { useEffect } from 'preact/hooks'
+import { useEffect } from 'react'
 import { useLocation } from 'wouter-preact'
-import { verificationFrom } from 'helpers/data'
 import AppMotto from 'icons/AppMotto'
-import Button from 'components/Button'
+import GradientSeparator from 'components/GradientSeparator'
+import MainCarousel from 'components/MainCarousel'
 import StoreButtons from 'components/StoreButtons'
 import WhatPeopleThink from 'icons/WhatPeopleThink'
 import classnames, {
   alignItems,
   display,
   flexDirection,
   gap,
-  margin,
   padding,
-  scale,
   transitionProperty,
   width,
   zIndex,
 } from 'classnames/tailwind'
 import getHashComponent from 'helpers/getHashComponent'
-import openBlankTab from 'helpers/openBlankTab'
+import useUserAgent from 'hooks/useUserAgent'
 
 const whatPeopleThinkWrapper = classnames(
-  scale('scale-50', 'xs:scale-65', 'md:scale-100'),
-  transitionProperty('transition-transform')
+  width('w-56', 'xs:w-72', 'md:w-96'),
+  transitionProperty('transition-transform'),
```

Comment:
You're not doing transitions for this block, no point in setting the property

Reply:
Changing width is not a transition if it's not animated (here we use breakpoints)
Previously this property was used for `scale` - which is transition

Reply:
I didn't use scale because it screwed up the margins, but I removed the transition.